### ISSUE TITLE
Cities/Towns list disappears on Search form

### ISSUE THREAD

**Issue by kkilbey** [BCcampus/eypd#643]
### Description 

On the Early Years home page, the City/Community/Town drop-down list disappears after changing from "All Provinces" to "British Columbia" and back to "All Provinces."

This is how the City/Town list displays initially:
![image](https://user-images.githubusercontent.com/44710357/51354481-97f75300-1a68-11e9-90a1-025dabb68136.png)

When I select British Columbia from the list of provinces, the list disappears (is this intentional?):
![image](https://user-images.githubusercontent.com/44710357/51354521-be1cf300-1a68-11e9-953c-b69dbcf4246e.png)

Then if I switch back to "All Provinces," the City/Town list is not restored:
![image](https://user-images.githubusercontent.com/44710357/51354561-de4cb200-1a68-11e9-8a21-21a42e640d3c.png)

Also note that Vancouver appears at the top of the list and alphabetically near the bottom. Does anyone happen to know why that is? (It is not the default.)

**Expected behaviour** 

The Cities/Towns list appears when "All Provinces" is selected, even after choosing "British Columbia" and then switching back to "All Provinces."

**Actual behaviour** 

The City/Towns list disappears when "All Provinces" is selected for a second time.

### Steps to reproduce the problem

1. Go to the Early Years home page
2. Click on the Cities/Communities/Towns drop-down list to display list of cities
3. Switch from "All Provinces" to "British Columbia"
4. Click on the Cities drop-down list again and see only one option: "All Cities/Towns"
5. Switch from "British Columbia" back to "All Provinces"
6. Click on the Cities drop-down list again and note that list of cities is not restored

**Comment by alex-418** [comment]
This has been resolved on cert and production. There was an update to the database structure on the WordPress side, which broke the plugins ability to create the relationships between provinces and towns(since it's a custom post type). All of our events are in BC, so I have disabled the Province dropdown list, and left the town field. It saves the user a step in searching, and resolves the issue. When the plugin updates to fix the issue, we can look at enabling this field again if it's needed.
